packages/bpk-component-label/src/BpkLabel.module.scss

```diff
@@ -31,6 +31,10 @@
 
   &--disabled {
     @include bpk-label--disabled;
+
+    &--white {
+      color: $bpk-text-disabled-night;
```

Comment:
Night tokens are only reserved for dark mode and shouldn't be used for Web whilst we don't support dark mode. We should check with design what we can use here

Reply:
That's what I thought too, but we have actually used night tokens for light mode in the past.
I'll check with design though

Reply:
Spoke to design and created new disabled on dark tokens!